presto-main/src/test/java/com/facebook/presto/type/TestRealOperators.java

```diff
@@ -186,6 +187,16 @@ public void testCastToVarchar()
     }
 
     @Test
+    public void testCastToChar()
+            throws Exception
+    {
+        assertFunction("CAST(REAL'754.1985' as CHAR)", CHAR, "754.1985");
```

Comment:
Sorry, the test should fail exactly. It seems that assertFunction in `Test*Operators` doesn't return actual value. Even if I write following test in `TestIntegerOperators`, the test throw exception. I look for the test place.

```java
    @Test
    public void testCastToVarchar()
            throws Exception
    {
        assertFunction("cast(INTEGER'49' as varchar(1))", VarcharType.createVarcharType(1), "4");
    }
```

```
java.lang.AssertionError: expected [4] but found [49]
Expected :4
Actual   :49
```